Sources/HummingbirdAuth/Authenticator/LoginCache.swift

```diff
@@ -28,34 +28,34 @@ public struct LoginCache: Sendable {
     }
 
     /// Logout authenticatable object. Removes object from cache
-    /// - Parameter auth: authentication type
+    /// - Parameter authenticatedType: authentication type
     @inlinable
-    public mutating func logout<Auth: Authenticatable>(_: Auth.Type) {
-        self.cache[ObjectIdentifier(Auth.self)] = nil
+    public mutating func logout<Auth: Authenticatable>(_ authenticatedType: Auth.Type) {
+        self.cache[ObjectIdentifier(authenticatedType)] = nil
```

Comment:
Why did you change this? IIRC passing `Auth.self` is more efficient than passing the metatype value.

Reply:
If it is quicker I'll revert. I did this to avoid an annoying swift format/documentation issue where swift format removes the parameter name and docc complains about the parameter name